### PR TITLE
Fix GitHub Actions errors

### PR DESCRIPTION
**Pull request by TGP17**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -70,13 +70,13 @@ jobs:
           CMAKE_C_COMPILER_LAUNCHER: "ccache"
           CMAKE_CXX_COMPILER_LAUNCHER: "ccache"
           CCACHE_COMPILERCHECK: "string:${{ env.NDK_VERSION }}"
-        run: ./gradlew --stacktrace --configuration-cache --build-cache --parallel --configure-on-demand assembleFullRelease assembleFullReldebug
+        run: ./gradlew --stacktrace --configuration-cache --build-cache --parallel --configure-on-demand assemblemainlineRelease assemblemainlineReldebug
```
Comment (lynxnb) [inline .github/workflows/ci.yml:73]: Are you sure this is not `assembleMainlineRelease` with the uppercase M?

**Review by lynxnb — CHANGES_REQUESTED** [review]
We do not want to skip git versioning for CI builds, as the app relies on it to show the version in the home screen.
The correct solution would be to disable configuration cache instead.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (Ishan09811) [reply .github/workflows/ci.yml:73]: @lynxnb 
I tested it its working fine 🙂

Comment (TGP17) [reply .github/workflows/ci.yml]: Should be fixed now, looks like build variants aren't case sensitive.

**Comment by TGP17** [comment]
> We do not want to skip git versioning for CI builds, as the app relies on it to show the version in the home screen. The correct solution would be to disable configuration cache instead.

Alright I disabled the configuration cache in the workflow file now, instead of skipping git versioning.

**Review by lynxnb — APPROVED** [review]
Looks good now, thanks!

**Comment by TGP17** [comment]
You're welcome!

**Pull request merged**